src/NGPL.xml

```diff
@@ -0,0 +1,95 @@
+<spdx name="Nethack General Public License" identifier="NGPL" osi-approved="true">
+  <urls>
+    <url>http://www.opensource.org/licenses/NGPL</url>
+  </urls>
+  <license>
+    <title>
+      <p>NETHACK GENERAL PUBLIC LICENSE</p>
+    </title>
+    <copyright>
+      <p>(Copyright 1989 M. Stephenson)</p>
+    </copyright>
+    <optional>
```

Comment:
Not sure this should be optional.  Perhaps this one is worthy of discussion on a call.